front/src/components/ContainerPage.tsx

```diff
@@ -0,0 +1,25 @@
+import { Typography, Container } from "@mui/material";
+import { ReactNode } from "react";
```

Comment:
```suggestion
import type { ReactNode } from "react";
```
prefer this so code is not added to compiled js output